playbooks/ps_81_upgrade.yml

```diff
@@ -0,0 +1,279 @@
+
+  - name: start my.cnf change test
+    command: /package-testing/check_mycnf_pre.sh
+
+#
+# Enable testing repository
+#
+  - name: include tasks for enabling PS 8.1 testing repo
+    include: ../tasks/enable_ps81_testing_repo.yml
+
+# Upgrade packages
+#
+  - name: upgrade Percona Toolkit to new deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: yes
+      state: latest
+    vars:
+      packages:
+      - percona-toolkit
+    when: ansible_os_family == "Debian"
+
+  - name: upgrade Percona Server to new deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: no
+      state: latest
+    vars:
+      packages:
+      - percona-server-server
+      - percona-server-test
+      - percona-server-client
+      - percona-server-dbg
+      - percona-server-source
+      - percona-server-rocksdb
+    when: ansible_os_family == "Debian"
+
+  - name: upgrade Percona XtraBackup to new deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: no
+      state: latest
+    vars:
+      packages:
+      - percona-xtrabackup-81
+      - percona-xtrabackup-test-81
+      - percona-xtrabackup-dbg-81
+    when: ansible_os_family == "Debian"
+
+  - name: upgrade Percona release to new rpm packages
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-release
+    when: ansible_os_family == "RedHat"
+
+  - name: upgrade Percona Server to new rpm packages on CentOS
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-server-server
+      - percona-server-test
+      - percona-server-debuginfo
+      - percona-server-devel
+      - percona-server-rocksdb
+      - percona-server-shared
+      - percona-server-shared-compat
+      - percona-server-client
+    when: ansible_os_family == "RedHat" and ansible_distribution_major_version != "9"
+
+  - name: upgrade Percona Server to new rpm packages on Oracle Linux 9
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-server-server
+      - percona-server-test
+      - percona-server-debuginfo
+      - percona-server-devel
+      - percona-server-rocksdb
+      - percona-server-shared
+      - percona-server-client
+    when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "9"
+
+  - name: upgrade Percona XtraBackup to new rpm packages
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-xtrabackup-81
+      - percona-xtrabackup-test-81
+      - percona-xtrabackup-81-debuginfo
+    when: ansible_os_family == "RedHat"
+
+  - name: install percona-mysql-shell package
+    include_tasks: ../tasks/install_pshell.yml
+    when: install_mysql_shell == "yes" or install_mysql_shell == ""
+
+  - name: list installed packages
+    include_tasks: ../tasks/list_installed_packages.yml
+
+  - name: check telemetry (enabled)
```

Comment:
Change to disabled:

name: check telemetry (disabled)
command: /package-testing/check_telemetry.sh ps -d